eventuate-tram-examples-common/src/main/java/io/eventuate/tram/examples/basic/events/DomainEventDispatcherInitializer.java

```diff
@@ -0,0 +1,19 @@
+package io.eventuate.tram.examples.basic.events;
+
+import io.eventuate.tram.events.subscriber.DomainEventDispatcher;
+import io.micronaut.context.annotation.Context;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+@Context
+public class DomainEventDispatcherInitializer {
```

Comment:
This should be moved into the same module as where the Micronaut bean for DomainEventDispatcherFactory is defined.

And generalized to have `DomainEventDispatcher[]`

Reply:
yes, I remember, will do